webpack.config.js

```diff
@@ -0,0 +1,28 @@
+var path = require('path');
+var webpack = webpack = require('webpack');
+
+var IS_PRODUCTION = process.env.NODE_ENV == 'production';
+
+module.exports = {
+  output: {
+    path: path.resolve(__dirname, "build", "web", "js"),
+    filename: "[name].js",
+  },
+  devtool: IS_PRODUCTION ? 'source-map' : 'inline-source-map',
+  entry: {
+    "beforePyret": './src/web/js/beforePyret.js',
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+      'process.env.PYRET': JSON.stringify(process.env.PYRET),
+      'process.env.CURRENT_PYRET_RELEASE': JSON.stringify(process.env.CURRENT_PYRET_RELEASE),
```

Comment:
It actually gets inlined, so you won't find `PYRET` in the build directory. Rather you will find whatever the value of `process.env.PYRET` was at build time.

This is especially useful when combined with dead code removal because you can do stuff like this:

```javascript
if (process.env.DEBUG) {
  console.log("this won't show up in production/minified js files");
}
```

and it gets turned into this:

```javascript
if (undefined) {
  console.log("this won't show up in production/minified js files");
}
```

which the static analyzer in the uglify plugin will recognize as dead code and completely remove.